node_cli/tasks.js

```diff
@@ -0,0 +1,167 @@
+const inquirer = require("inquirer");
+const Realm = require("realm");
+const mongodb = require("mongodb");
+const index = require("./index");
+
+exports.getTasks = async () => {
+  const realm = await index.getRealm();
+  console.dir(realm);
+  const tasks = await realm.objects("Task");
+  index.output("MY TASKS:", "header");
+  index.output(JSON.stringify(tasks, null, 3), "result");
+};
+
+exports.getTask = async () => {
+  const realm = await index.getRealm();
+  try {
+    const task = await inquirer.prompt([
+      {
+        type: "input",
+        name: "id",
+        message: "What is the task ID (_id)?",
+      },
+    ]);
+    let result = await realm.objectForPrimaryKey(
+      "Task",
+      mongodb.ObjectID(task.id)
+    );
+    if (result !== undefined) {
+      index.output("Here is the task you requested:", "header");
+      index.output(JSON.stringify(result, " ", 3), "result");
+    }
+  } catch (err) {
+    index.output(err, "error");
+  }
+};
+
+exports.createTask = async () => {
+  const realm = await index.getRealm();
+  try {
+    index.output("*** CREATE NEW TASK ***", "header");
+    const task = await inquirer.prompt([
+      {
+        type: "input",
+        name: "name",
+        message: "What is the task text?",
+      },
+      {
+        type: "rawlist",
+        name: "status",
+        message: "What is the task status?",
+        choices: ["Open", "In Progress", "Closed"],
+        default: function () {
+          return "Open";
+        },
+      },
+    ]);
+    let result;
+    realm.write(async () => {
```

Comment:
The write callbacks can't be async, but realm.create() is not async and does not need to be awaited.